cypress/e2e/dashboard/events/[eventId].cy.js

```diff
@@ -102,5 +102,35 @@ describe('event dashboard', () => {
         cy.get('@rsvps').contains(userName);
       });
     });
+
+    it('prevents members from confirming or kicking users', () => {
+      const eventId = 1;
+      // Start as the instance owner to ensure we can find the RSVPs
+      cy.login();
```

Comment:
Not sure if this is needed explicitly, as `beforeEach` logs in as owner, comment might be enough. :man_shrugging: